src/utils/parse.ts

```diff
@@ -29,12 +29,12 @@ const html = htm.bind((type: string, props, ...children) => {
     newProps.style = objStyle
   }
 
-  return [type, newProps, children.map(c => decodeEntities(c))]
+  return [type, newProps, children.map((c) => decodeEntities(c))]
 })
 
 export default function parse(htmlStr: string) {
   const lines = htmlStr.split('\n')
   const breaks = [...Array(lines.length - 1)].map(() => '\n')
 
-  return html(lines, ...breaks)
+  return html((lines as any) as TemplateStringsArray, ...breaks)
```

Comment:
Hm, I cloud not deal with the type error without `(lines as any) as TemplateStringsArray`.
Please advise me if you know a better solution 🙇‍♂️